packages/notification-center/src/shared/enums/index.ts

```diff
@@ -0,0 +1,3 @@
+export { ChannelTypeEnum, ChannelCTATypeEnum, MessageActionStatusEnum, ButtonTypeEnum } from '@novu/shared';
```

Comment:
Added more enums and interfaces for the user to use. so they wont need to extract it from two packages (@shared in addition)